### PR TITLE
[Bug][Seatunnel-web] Task instance filter is not working as expected

### PR DESCRIPTION
**Pull request by arshadmohammad**
Corredcted task instance filter logic
Changed logback version to 1.2.3

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by arshadmohammad** [comment]
Modified existing test case to verify the filter scenario. executed all integration test cases
image

**Comment by arshadmohammad** [comment]
After fix, filter is working as expected
image

**Comment by Hisoka-X** [comment]
Please fix ci.

**Comment by arshadmohammad** [comment]
Seems PR CI is failed because of CI environment issue
`Error:  Failed to execute goal on project seatunnel-dynamicform: Could not resolve dependencies for project org.apache.seatunnel:seatunnel-dynamicform:jar:1.0.2-SNAPSHOT: Could not transfer artifact com.fasterxml.jackson.core:jackson-databind:jar:2.12.6 from/to central (https://repo.maven.apache.org/maven2): /home/runner/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.12.6/jackson-databind-2.12.6.jar.part (No such file or directory)`

Let me retrigger the PR CI by pushing dummy change.

**Comment by arshadmohammad** [comment]
> Backend / Integration Test (11, ubuntu-latest) (pull_request) Failed 

Error:  /home/runner/work/seatunnel-web/seatunnel-web/seatunnel-server/seatunnel-server-common/src/main/java/org/apache/seatunnel/server/common/CodeGenerateUtils.java:[22,12] cannot access java.util
Error:    ZipException opening "seatunnel-jackson-2.3.7-optional.jar": zip END header not found

**Comment by arshadmohammad** [comment]
Finally build is success.

File: pom.xml [hunk 1]
```diff
@@ -105,7 +105,7 @@
         <hadoop-aws.version>3.1.4</hadoop-aws.version>
         <aws-java-sdk-bundle.version>1.11.271</aws-java-sdk-bundle.version>
         <spotless.version>2.29.0</spotless.version>
-        <logback.version>1.5.7</logback.version>
+        <logback.version>1.2.3</logback.version>
```
Comment (Hisoka-X) [inline pom.xml:108]: Why we should downgrade logback version?

Comment (arshadmohammad) [reply pom.xml:108]: 1. Despite upgrading to version 1.5.7, there remains an unresolved Common Vulnerabilities and Exposures (CVE) issue in this version.
2. Upon upgrading to 1.5.7, the log4j-slf4j-impl jar file is absent from the project, resulting in failed logging attempts.

So reverted as of now. Will check more on this later

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @arshadmohammad

**Comment by arshadmohammad** [comment]
Thanks @Hisoka-X for reviewing and merging the PR.

**Pull request merged**